app/benchmarks/benchmark_msg_send_test.go

```diff
@@ -307,14 +307,17 @@ func generateMsgSendTransactions(b *testing.B, count int) (*app.App, [][]byte) {
 	return testApp, rawTxs
 }
 
+// megabyte the number of bytes in a megabyte
+const megabyte = 1048576
```

Comment:
[uber nit] this is the number of bytes in a mebibyte 

```suggestion
// mebibyte the number of bytes in a mebibyte
const mebibyte = 1048576
```

![Screenshot 2024-10-20 at 9 47 55 PM](https://github.com/user-attachments/assets/6fd10112-5d63-41fe-bbfd-5bbbdd27e943)
![Screenshot 2024-10-20 at 9 47 47 PM](https://github.com/user-attachments/assets/4da9a5da-20b2-4780-9111-53e581abd849)